src/main/java/spoon/reflect/declaration/CtModuleMember.java

```diff
@@ -0,0 +1,9 @@
+package spoon.reflect.declaration;
+
+/**
+ * Represents a component from a module
+ */
+public interface CtModuleMember extends CtElement {
```

Comment:
Note: the java specification calls it `ModuleDirective`. So why we invent new name `ModuleMember`? I personally have no problem with both names, so if You think ModuleMember is good one. I will like it too ;-)

Reply:
agree